src/utils/constants.ts

```diff
@@ -1,6 +1,6 @@
 // Minimum eight characters, at least one uppercase letter,
 // one lowercase letter, one number and one special character
-export const PASSWORD_REGEX = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/;
+export const PASSWORD_REGEX = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,255}$/;
```

Comment:
Password is limited to 72 characters, as per bcrypt documentation. Regexes should not enforce size, as it is the responsibility of DTO validators.

Reply:
Noted. I'll change the maximum size on frontend too then.